updateJsonSchema.js

```diff
@@ -0,0 +1,38 @@
+const offer_contents = (await OfferContent.find())
+
+async function updatePolls () {
+	for (let i = 0; i < offer_contents.length; i++) {
+		const oc = offer_contents[i];
+		const pageTypeNum = 0;
+		if (oc.contentType !== 'poll') { continue; }; // I divided polls and quizzes to two separate calls
+		const content = JSON.parse(oc.content);
+		content.pages.forEach(page => {
+			if (page.type !== pageTypeNum) { return false; }
+			page.rewardText = page.title.replace(/ \${amount}.*/, '');
+			page.rewardValue = '${amount}';
+			page.title = '';
```

Comment:
Is title still used anywhere?

Reply:
Every poll/quiz in `test` has title. In some of them it's empty string. So I put here an empty string as well